docs/Configuration/AdvancedMode.md

```diff
@@ -325,6 +326,27 @@ Below is an example for collecting a bitbucket repo.
 - `owner`: the owner of the repository.
 - `repo`: the bitbucket repository name.
 
+### 10. Sonarqube
+
+Below is an example for collecting a sonarqube project.
+
+```json
+[
+  [
+    {
+      "plugin": "sonarqube",
+      "options": {
+        "connectionId": 1,
+        "projectKey": "testDevLake"
+      }
+    }
+  ]
+]
+```
+
+- `connectionId`: The ID of your sonarqube connection at page http://localhost:4000/connections/sonarqube.
+- `projectKey`: the project name of the sonarqube.
```

Comment:
ProjectKey or ProjectName? If it is ProjectKey, please show users how to get it with a URL to the SonarQube doc or a screenshot.